Assignment2/FRM95-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,59 @@
+{
+   "@context":{
+   "schema":"https://schema.org",
+   "age":"ttp://xmlns.com/foaf/0.1/age",
+   "subjects":"http://www.w3.org/2001/fiupm/asignaturas/",
+   "name":"schema:name",
+   "birthPlace":"schema:birthPlace",
+   "parents":"schema:parents",
+   "father":"parents:father",
+   "mother":"parents:mother",
+   "alumniOf":"schema:alumniOf",
+   "almaMater":"alumniOf:name",
+   "url":"alumniOf:url",
```

Comment:
These attribute definitions work, but semantically they aren't ok. The best way to define reusing your prefix schema and name it properly e.g. "father":"schema:father".
You don't need to change them, but remember to provide good definitions for the future.